packages/ui/components/card/Card.tsx

```diff
@@ -148,12 +148,13 @@ export function Card({
               onClick={actionButton?.onClick}
               target="_blank"
               rel="noreferrer"
-              className="text-xs font-medium">
+              prefetch={!learnMore.href.startsWith("https://cal.com/blog")}
```

Comment:
Is this prefect coming from this PR or is it reintroducing previously removed code?

Reply:
I think it got removed in a recent PR and being accidentally added here.